code/modules/jobs/jobs.dm

```diff
@@ -60,6 +60,18 @@ GLOBAL_LIST_INIT(nonhuman_positions, list(
 	"Cyborg",
 	ROLE_PAI))
 
+// job categories for rendering the late join menu
+GLOBAL_LIST_INIT(position_categories, list(
+	EXP_TYPE_COMMAND = list("jobs" = command_positions, "color" = "#ccccff"),
+	EXP_TYPE_ENGINEERING = list("jobs" = engineering_positions, "color" = "#ffeeaa"),
+	EXP_TYPE_SUPPLY = list("jobs" = supply_positions, "color" = "#ddddff"),
+	EXP_TYPE_SILICON = list("jobs" = GLOB.nonhuman_positions - "pAI", "color" = "#ccffcc"),

```

Comment:
Chop the glob here. Since it's all in context of global ss anyway.

Reply:
Whoops, good ol copypastes. Fixed.